src/components/MovieDetails/MovieDetails.js

```diff
@@ -2,11 +2,68 @@ import React from 'react';
 import './MovieDetails.scss';
 import { trailers } from '../../trailers';
 
-const MovieDetails = ({ location }) => {
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { setMovies, setRatings } from '../../actions/actions';
+import { addUserRating, getRatings, deleteRating } from '../../apiCalls';
+
+
+export const MovieDetails = ({ location, user, ratings, setRatings }) => {
+
+  const removeRating = async (event) => {
+    let ratingID = event.target.id;
+    await deleteRating(user.id, ratingID)
+    await getSetRatings()
+  }
+
+  const checkIfRated = (ratings, movie) => {
+    let movieRating = ratings.find(rating => {
+      return rating.movie_id === movie.id
+    })
+    if(movieRating) {
+      return <div>
+          <h3 className='rating-label'>Your rating: {movieRating.rating}</h3>
+          <button id={movieRating.id} onClick={(e) => removeRating(e)}>Change rating</button>
+        </div>
+    } else {
+      return <div className='user-rating'>
+          <h3 className='rating-label'>Your rating:</h3>
+          <select id={movie.id} className='rating-dropdown' onChange={ (e) => {submitUserRating(e)}}>
```

Comment:
My comment about the `onClick` can be applied to the `onChange` here. Also, you have a straggler space between the `{` and `(` in your `onChange`. Linting will catch these things for you, so implement that ASAP.